.gitignore

```diff
@@ -0,0 +1,3 @@
+pos-backend/node_modules/
+pos-backend/.env
```

Comment:
why have you added env file in gitignore ?

Reply:
Yes